parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -60,7 +60,7 @@ class ReadConf<T> {
   private final List<Map<ColumnPath, ColumnChunkMetaData>> columnChunkMetaDataForRowGroups;
 
   @SuppressWarnings("unchecked")
-  ReadConf(
+  public ReadConf(
```

Comment:
do we need this public?

Reply:
Yes we do. The Comet reader is now in a different module and needs to access the read conf for read information. I could make it protected and then derive a class explicitly meant for custom readers, but that seems overkill.

Reply:
I would rather not rename an existing class. I feel it will reduce confusion on the scope of the changes.